doc/api/http.md

```diff
@@ -1987,6 +1989,49 @@ header will not yield the expected result. If progressive population of headers
 is desired with potential future retrieval and modification, use
 [`response.setHeader()`][] instead of [`response.writeHead()`][].
 
+### `response.setHeaders(headers)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `headers` {Object|Array}
```

Comment:
The whole point of this method is to accept a `Headers` object, not Object or Array.

Reply:
@mcollina can we accept both object/array and Headers and also add Headers to `writeHead`?

Reply:
No, this is specific to `Headers`.